src/appMain/sdl_preloaded_pt.json

```diff
@@ -3541,7 +3541,7 @@
                             "name": "pressureTelltale",
                             "key": "OEM_REF_PRES_TEL",
                             "type": "WarningLightStatus",
-                            "mandatory": true
+                            "mandatory": false
```

Comment:
Can you please add `deprecated`, `since`, `until` fields as well for all below params. See below for an example:
https://github.com/smartdevicelink/sdl_core/blob/61ac5345cd9564605f3f7f46170c773db3f9ab74/src/appMain/sdl_preloaded_pt.json#L3129

https://github.com/smartdevicelink/sdl_core/blob/61ac5345cd9564605f3f7f46170c773db3f9ab74/src/appMain/sdl_preloaded_pt.json#L3410

Reply:
I believe that we need only "since" parameter here, please check 
2ff2c61

Reply:
@LitvinenkoIra  - We'd need `since` and `until` to maintain the history of the param. And then another entry with the new param with just `since`. See example for `utcYear`: https://github.com/smartdevicelink/sdl_core/blob/61ac5345cd9564605f3f7f46170c773db3f9ab74/src/appMain/sdl_preloaded_pt.json#L3076
&
https://github.com/smartdevicelink/sdl_core/blob/61ac5345cd9564605f3f7f46170c773db3f9ab74/src/appMain/sdl_preloaded_pt.json#L3085

Reply:
@atiwari9 Got it, thank you, please check https://github.com/smartdevicelink/sdl_core/pull/3583/commits/2ff1d2cc28d7151bdaf711e69502e0484e6cfaa9